homeassistant/components/toon/switch.py

```diff
@@ -59,8 +59,6 @@ def __init__(self, coordinator: ToonDataUpdateCoordinator, *, key: str) -> None:
     def unique_id(self) -> str:
         """Return the unique ID for this binary sensor."""
         agreement_id = self.coordinator.data.agreement.agreement_id
-        # This unique ID is a bit ugly and contains unneeded information.
-        # It is here for legacy / backward compatible reasons.
         return f"{DOMAIN}_{agreement_id}_switch_{self.key}"
```

Comment:
We don't need `DOMAIN` or `switch` in here. The entity registry is aware of both the integration domain and the platform domain.

Reply:
Right, since these are new, that will work, thanks 👍